### PR TITLE
fix: Resolves issues with server not responding to SQP

### PR DESCRIPTION
**Pull request by danmrichards**
Resolves an issue where the SQP handler would not respond to queries. This resulted in Multiplay killing servers prematurely

### CODE REVIEW

File: Assets/Scripts/Matchplay/Server/ServerGameManager.cs [hunk 1]
```diff
@@ -35,7 +36,7 @@ public ServerGameManager(string serverIP, int serverPort, int serverQPort, Netwo
             m_ServerPort = serverPort;
             m_QueryPort = serverQPort;
             m_NetworkServer = new MatchplayNetworkServer(manager);
-            m_MultiplayAllocationService = new MultiplayAllocationService();
```
[the remark this answers is not in the document]
Comment (UnityJacob) [reply Assets/Scripts/Matchplay/Server/ServerGameManager.cs:38]: I'll try a refactored version that removes this class entirely and see if it still runs. As it stands this wont be able to receive Matchmaker Payloads to configure the game.

**Review by UnityJacob — COMMENTED** [review]
I agree with the renaming and Its awesome to fix the Bug. 
I think the MultiplayAllocationService class is depricated and we should move its function into the SQPService class.

File: Assets/Scripts/Matchplay/Server/ServerGameManager.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (danmrichards) [reply Assets/Scripts/Matchplay/Server/ServerGameManager.cs:38]: Good catch, that's a typo on my part. Was supposed to just be adding the query service, not removing the alloc service as well.

I would prefer we have two separate classes for this. Allocation and server query are two separate APIs, the logic for them should not be coupled together. Game servers should run server query irrespective of allocations, this was the confusion that cause the original bug here.

Lets chat offline about this too

[the remark this answers is not in the document]
Comment (danmrichards) [reply Assets/Scripts/Matchplay/Server/ServerGameManager.cs:38]: As we discussed in Slack, I restored the missing line

**Review by UnityJacob — APPROVED** [review]
LGTM : Still works locally and with matchmaker online.

**Pull request merged**